src/types/chat.ts

```diff
@@ -13,17 +13,19 @@ export enum ChatMessageType {
   poll = "poll",
 }
 
+export type ChatUser = Pick<User, "partyName" | "pictureUrl" | "anonMode">;
+
 export type BaseChatMessage = {
-  from: string;
+  from: WithId<ChatUser>;
```

Comment:
I lean towards renaming it to `fromUser` and `toUser`. Since we already change the structure. + rename ts_utc to timestamp

Reply:
renamed in 

- https://github.com/sparkletown/sparkle/pull/2304/commits/20457ad6ae3e423d0d2f636d087d6413aee448d7
- https://github.com/sparkletown/sparkle/pull/2304/commits/440a95f3fbb83da3aa5d0029fa96ef6160913e76
- https://github.com/sparkletown/sparkle/pull/2304/commits/b5d5005b7fe96d278b9f2d76a25ea4f23acb20fb